File: Exercícios de Lógica/Java/Extra/Exrcicios_Java/src/Exercicio19.java
/*
Estruturas de decisão
Exercicio 019
Faça um Programa que peça dois números e imprima o maior deles.

 */

import java.util.Scanner;
public class Exercicio19 {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        double num1 = scanner.nextDouble();

        System.out.print("Digite o segundo número: ");
        double num2 = scanner.nextDouble();

        if (num1>num2){
        System.out.print(num1+ " é o maior número");
    }
        else if(num2>num1){
            System.out.print(num2+ " é o maior número");
        }
        else{
            System.out.print("Os dois números são iguais");
        }
        scanner.close();
        }

}

File: Exercícios de Lógica/Python/Exercicio19.py
'''
Exercicio 019

Faça um Programa que peça dois números e imprima o maior deles.
'''

num1 = int(input("Digite o primeiro número: "))
num2 = int(input("Digite o segundo número: "))

if num1>num2:
    print("O primeiro número é o maior.")

elif num1<num2:
    print("O segundo número é o maior.")

else:
    print("Os dois números são iguais")